extension/content.js

```diff
@@ -220,6 +220,46 @@ function showRecentlyPushedBranches() {
 	$('.repository-content').prepend(div);
 }
 
+// Add option for viewing diffs without whitespace changes
+function addDiffViewWithoutWhitespaceOption() {
+	if ($('.diff-options-content').length < 1 && $('.btn-group .selected[href*="diff="]').length < 1) {
+		return;
+	}
+
+	// Detect if PR or commit
```

Comment:
We have utility functions for this already in https://github.com/sindresorhus/refined-github/blob/master/extension/page-detect.js